events/wc_dark_cult_events.txt

```diff
@@ -1050,62 +1096,179 @@ narrative_event = {
 	immediate = {
 		# Used in scripted effects
 		save_event_target_as = target_settler
-		
+
 		religion = old_gods_worship
+		add_trait = zealous
 		clear_event_target = yes
-		
+		any_courtier = {
+			limit = { culture = twilights_hammer }
+			religion = old_gods_worship
+		}
+
 		remove_trait = travelling
-		
+
+
 		c_hivezora = {
 			# event_target:target_settler and its people settle here
 			settle_as_target_settler_effect = yes
 		}
-		
+
 		# If you're not independent ruler, gives old titles to liege
 		give_old_titles_to_heir_effect = yes
-		
-		d_crystalvale = {
-			any_direct_de_jure_vassal_title = {
-				# event_target:target_settler and its people settle here
-				settle_as_target_settler_effect = yes
-			}
+
+		c_hiveashi = {
+			# event_target:target_settler and its people settle here
+			settle_as_target_settler_effect = yes
 		}
-		d_scarab_wall = {
-			any_direct_de_jure_vassal_title = {
-				# event_target:target_settler and its people settle here
-				settle_as_target_settler_effect = yes
-			}
+		c_crystalvale = {
+			settle_as_target_settler_effect = yes
+		}
+		c_ishrak = {
+			settle_as_target_settler_effect = yes
+		}
+		c_ashakx = {
+			settle_as_target_settler_effect = yes
+		}
+		c_hiveregal = {
+			settle_as_target_settler_effect = yes
+		}
+		c_scarab_wall = {
+			settle_as_target_settler_effect = yes
 		}
-		
 		c_hivezora = {
 			# Declares war against player
 			target_settler_declare_war_effect = yes
 		}
-		
+
 		clear_settler_flag_effect = yes
-		
-		# Spawns army for start
-		spawn_colonial_troops_effect = yes
+		c_hivezora = { location = { culture = qiraji } }
+		c_hiveashi = { location = { culture = qiraji } }
+		c_crystalvale = { location = { culture = qiraji } }
+		c_ishrak = { location = { culture = qiraji } }
+		c_ashakx = { location = { culture = qiraji } }
+		c_hiveregal = { location = { culture = qiraji } }
+		c_scarab_wall = { location = { culture = qiraji } }
 	}
 
 	# I'll spread its will across Kalimdor!
 	option = {
 		name = EVTOPTA_WCDAC_5040
+		if = {
+			limit = { has_dlc = "Mystics" }
+			leave_society = yes
+			set_global_flag = twilights_hammer_cult_active_flag
+			join_society_twilights_hammer_cult_effect = yes
+			hidden_effect = {
+				set_society_grandmaster = yes # Set character as Grand Master
+				add_society_currency_massive_effect = yes # Give the Grand Master a start-up capital to induct members
+			}
+		}
+		if = {
+			limit = {
+				is_shadow_priest_class_trigger = no
+				character = 52000
+			}
```

Comment:
Should check `potential_shadow_priest_wo_training_trigger`.

Reply:
he starts with magic class and he gets his religion set in the same event

Reply:
And? Does this block him from having `potential_shadow_priest_wo_training_trigger`?